src/services/room.service.js

```diff
@@ -0,0 +1,47 @@
+const { ApiError } = require('../exceptions/api.error');
+const { Room } = require('../models/Room.model');
+
+const create = async (name) => {
+  if (!name) {
+    throw ApiError.badRequest('name is required');
+  }
+
+  const newRoom = await Room.create({ name });
+
+  return newRoom;
+};
+
+const remove = async (id) => {
+  if (!id) {
+    throw ApiError.badRequest('id is required');
+  }
+
+  await Room.destroy({ where: { id } });
```

Comment:
Consider checking if the room with the given `id` exists before attempting to delete it. If it doesn't exist, you might want to throw an `ApiError.notFound()` to inform the caller that the room was not found.